src/test/kotlin/build/buf/gradle/LintWithWorkspaceTest.kt

```diff
@@ -21,4 +21,34 @@ class LintWithWorkspaceTest : NonProtobufGradlePluginLintTests, AbstractLintTest
     fun `lint with a config in workspace`() {
         assertSuccess()
     }
+
+    @Test
+    fun `lint a basic correct message with default config v2`() {
+        super.`lint a basic correct message with default config`()
```

Comment:
Heh. Do you have plans to hoist this up into the base test classes?

Reply:
What would you suggest as a good idiom for two tests that do exactly the same thing but use different resource dirs?

Reply:
I only added a `_v2` test for cases that contain a `buf.work.yaml` file, which are generally the `WithWorkspace` tests. Accordingly I define these cases only in the leaf classes.

Reply:
The `super` function is itself a `@Test` case that runs against the resource dir that does not end with `_v2`

Reply:
Right - and would it also make sense to run `v2` tests against a "vanilla" project (i.e., one not using workspaces)?

Reply:
I don't think we need to test against every version of Buf CLI - apart from some edge cases which we unfortunately ran into, it is meant to be fully backwards compatible.